zendesk/provider/client.py

```diff
@@ -0,0 +1,43 @@
+import base64
+import requests
+from flask import current_app as app
+
+from . import UpstreamProviderError
+
+AUTHORIZATION_HEADER = "Authorization"
+BEARER_PREFIX = "Bearer "
```

Comment:
Don't need these right? Since we aren't fetching the access token from the headers

Reply:
Thank you - fixed